### PR TITLE
:sparkles:   Switch zap log for global hub manager 

### PR DESCRIPTION
**Pull request by yanmxa**
## Related issue(s)

Fixes # https://issues.redhat.com/browse/ACM-13421, https://github.com/stolostron/multicluster-global-hub/issues/1178

## Tests
* [ ] Unit/function tests have been added and incorporated into `make unit-tests`.
* [ ] Integration tests have been added and incorporated into `make integration-test`.
* [ ] E2E tests have been added and incorporated into `make e2e-test-all`.
* [x] List other manual tests you have done.

```bash
2024-10-30T01:20:44.725Z        INFO    utils/utils.go:28       Go OS/Arch: linux/amd64
2024-10-30T01:20:44.725Z        INFO    utils/utils.go:29       Go Version: go1.22.5
2024-10-30T01:20:44.725Z        INFO    utils/utils.go:30       Git Commit: 0e0ccfa9
2024-10-30T01:20:44.744Z        INFO    cronjob/scheduler.go:68 set SyncLocalCompliance job     {"scheduleAt": "00:00"}
2024-10-30T01:20:44.744Z        INFO    cronjob/scheduler.go:77 set DataRetention jobscheduleAt00:00
2024-10-30T01:20:44.744Z        INFO    controller-runtime.metrics      server/server.go:208    Starting metrics server
I1030 01:20:44.744263       1 leaderelection.go:254] attempting to acquire leader lease multicluster-global-hub/multicluster-global-hub-manager-lock...
2024-10-30T01:20:44.744Z        INFO    controller-runtime.metrics      server/server.go:247    Serving metrics server  {"bindAddress": "0.0.0.0:8384", "secure": false}
I1030 01:23:06.543481       1 leaderelection.go:268] successfully acquired lease multicluster-global-hub/multicluster-global-hub-manager-lock
2024-10-30T01:23:06.543Z        INFO    cronjob/scheduler.go:86 start job scheduler
"ConfigMap", "worker count": 1}
2024-10-30T01:23:06.745Z        INFO    controller/controller.go:217    Starting workers        {"controller": "secret", "controllerGroup": "", "controllerKind": "Secret", "worker count": 1}
2024-10-30T01:23:06.745Z        INFO    controller/controller.go:217    Starting workers        {"controller": "backupPvcController", "controllerGroup": "", "controllerKind": "PersistentVolumeClaim", "worker count": 1}
2024-10-30T01:23:06.745Z        INFO    kafka-consumer  consumer/generic_consumer.go:84 transport consumer with cloudevents-kafka receiver
2024-10-30T01:23:06.750Z        INFO    dispatcher/transport_dispatcher.go:42   transport dispatcher starts dispatching received events...
2024-10-30T01:23:06.750Z        INFO    statistics      statistics/statistics.go:97     starting statistics
2024-10-30T01:23:06.750Z        INFO    dispatcher/conflation_dispatcher.go:64  starting dispatcher
2024-10-30T01:23:06.750Z        INFO    worker-pool     workerpool/worker_pool.go:38    connection stats        {"open connection(worker)": 1, "max": 10}
2024-10-30T01:23:06.750Z        INFO    worker-pool     workerpool/worker.go:46 started worker  {"WorkerID": 10}
...
2024-10-30T01:23:06.750Z        INFO    worker-pool     workerpool/worker.go:46 started worker  {"WorkerID": 5}
2024-10-30T01:23:06.750Z        INFO    worker-pool     workerpool/worker.go:46 started worker  {"WorkerID": 7}
2024-10-30T01:23:06.750Z        INFO    hub-management  hubmanagement/hub_management.go:73      hub management status switch frequency  {"interval": "2m0s"}
2024-10-30T01:23:06.750Z        INFO    controller/controller.go:175    Starting EventSource    {"controller": "migration-controller", "controllerGroup": "global-hub.open-cluster-management.io", "controllerKind": "ManagedClusterMigration", "source": "kind source: *v1alpha1.ManagedClusterMigration"}
2024-10-30T01:23:06.750Z        INFO    controller/controller.go:175    Starting EventSource    {"controller": "migration-controller", "controllerGroup": "global-hub.open-cluster-management.io", "controllerKind": "ManagedClusterMigration", "source": "kind source: *v1beta1.ManagedServiceAccount"}
2024-10-30T01:23:06.750Z        INFO    controller/controller.go:183    Starting Controller     {"controller": "migration-controller", "controllerGroup": "global-hub.open-cluster-management.io", "controllerKind": "ManagedClusterMigration"}
2024-10-30T01:23:06.754Z        INFO    kafka-consumer  consumer/generic_consumer.go:150        init consumeroffsets[]
2024-10-30T01:23:06.754Z        INFO    cloudevents     v2@v2.0.0-20240911135016-682f3a9684e4/protocol.go:166   Subscribing to topics: [^gh-status.*]
2024-10-30T01:23:06.853Z        INFO    controller/controller.go:217    Starting workers        {"controller": "migration-controller", "controllerGroup": "global-hub.open-cluster-management.io", "controllerKind": "ManagedClusterMigration", "worker count": 1}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.policy.localspec    conflator/element_complete.go:66        resetting element processed version     {"version": "0.1"}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.managedhub.heartbeat        conflator/element_complete.go:66        resetting element processed version     {"version": "0.0"}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.policy.localcompletecompliance      conflator/element_complete.go:66        resetting element processed version     {"version": "0.1"}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.policy.localcompliance      conflator/element_complete.go:66        resetting element processed version     {"version": "0.1"}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.managedcluster      conflator/element_complete.go:66        resetting element processed version     {"version": "0.1"}
2024-10-30T01:23:10.007Z        INFO    demo-hub-a.complete.managedhub.info     conflator/element_complete.go:66        resetting element processed version     {"version": "0.2"}
```

### CODE REVIEW

**Comment by yanmxa** [comment]
e2e setup error 
/test test-e2e

File: manager/cmd/main.go [hunk 1]
```diff
@@ -265,24 +261,22 @@ func transportCallback(mgr ctrl.Manager, managerConfig *configs.ManagerConfig) c
...
-		return 1
+		return fmt.Errorf("failed to complete configuration %w", err)
 	}
 
 	if managerConfig.EnablePprof {
 		go utils.StartDefaultPprofServer()
 	}
 
-	utils.PrintVersion(setupLog)
+	utils.RuntimeInfo()
```
Comment (clyang82) [inline manager/cmd/main.go:279]: maybe `PrintRuntimeInfo`?

File: operator/pkg/controllers/hubofhubs/manager/manifests/transport-config-secret.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@ kind: Secret
 metadata:
   name: {{.TransportConfigSecret}}
   namespace: {{.Namespace}}
+  annotations:
+    # todo put here or configmap
+    za-log-level: {{.LogLevel}}
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manager/manifests/transport-config-secret.yaml:8]: debug code?

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: operator/pkg/controllers/hubofhubs/manager/manifests/transport-config-secret.yaml (hunk 2 above)
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/manager/manifests/transport-config-secret.yaml:8]: shouldn't add in here, remove it now.

File: manager/cmd/main.go (hunk 1 above)
Comment (yanmxa) [reply manager/cmd/main.go:279]: Done.

**Pull request merged**